tests/test_ecs.py

```diff
@@ -100,6 +100,43 @@ def test_ecs_service_metrics(self):
         self.assertEqual(len(resources), 1)
         self.assertTrue("c7n.metrics" in resources[0])
 
+    def test_ecs_service_update(self):
+        session_factory = self.replay_flight_data("test_ecs_service_update")
+        test_service_name = 'custodian-service-update-test'
+
+        p = self.load_policy(
+            {
+                "name": "all-ecs-to-update",
+                "resource": "ecs-service",
+                "filters": [
+                    {"networkConfiguration.awsvpcConfiguration.assignPublicIp": "ENABLED"},
+                    {"serviceName": "custodian-test-service"}
+                ],
+                "actions": [
+                    {
+                        'type': 'modify',
+                        'update': {
+                            'networkConfiguration': {
+                                'awsvpcConfiguration': {
+                                    'assignPublicIp': 'DISABLED',
+                                    'securityGroups': ['sg-0669346d29dd0345a'],
+                                    'subnets': ['subnet-037318c577d65d330']
+                                }
+                            },
+                        }
+                    }
+                ],
+            })
```

Comment:
You need to pass the replay/record session factory in, and to assert a count on the policy run result. Atm, none of the code in  the action is being run in ci as a result.